pkgs/development/python-modules/phply/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ply
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "phply";
+  version = "1.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Cyd3TShfUHo0RYBaBfj7KZj1bXCScPeLiSCLZbDYSRc=";
+  };
+
+  dependencies = [
+    ply
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "phply" ];
+
+  meta = with lib; {
+    description = "Lexer and parser for PHP source implemented using PLY";
+    homepage = "https://github.com/viraptor/phply";
+    license = licenses.gpl2;
```

Comment:
```suggestion
    license = licenses.bsd3;
```

https://github.com/viraptor/phply/blob/master/LICENSE looks more like BSD, I guess?

Reply:
Oops, this was an oversight :grimacing: 
Looks like licenses.bsd3 to me, do you agree?

Reply:
I agree.